code/common/finspace.q

```diff
@@ -39,8 +39,8 @@ notifyhdb:{[cluster;changeset]
       // Ensuring that the changeset has successfully created before doing the HDB reload
       current:.finspace.checkstatus[(`.aws.get_changeset;.finspace.database;changeset[`id]);("COMPLETED";"FAILED");00:01;0wu];
       .lg.o[`notifyhdb;("notifying ",string[cluster]," to repoint to changeset ",changeset[`id])];
-      awsdb:.aws.db[.finspace.database;changeset[`id];.aws.cache["CACHE_1000";"/"];""];
-      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep["NO_RESTART"]]
+      awsdb:.aws.db[.finspace.database;changeset[`id];();".finspace.dataview"];
```

Comment:
I think you don't need the quotations around .finspace.database. 

Also maybe if we are unconcluded about whether we will support NO_RESTARTS, it may be preferable to encapsulate this in an if-else. Here's what I've been working with for example:

```
      usecache:"/opt/kx/app/scratch"~getenv[`KDBSCRATCH];
      awsdb:$[usecache;
          .aws.db[.finspace.database;changeset[`id];.aws.cache["CACHE_1000";"/"];""];
          .aws.db[.finspace.database;changeset[`id];();.finspace.dataview]
        ];
      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep $[usecache;"NO_RESTART";"ROLLING"]]
```

And just locally change the useCache flag locally depending on your use-case